### DISCUSSION TITLE
[auth0] missing parameter for ResetPasswordEmailOptions

### DISCUSSION THREAD

**Discussion by saewitz** [DefinitelyTyped/DefinitelyTyped#58553] [Issues with a @types package] [answered]
Hi, I think [the Auth0 definition for `ResetPasswordEmailOptions`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c319a1c881c1c76056713bcf1a8f124c6709132d/types/auth0/index.d.ts#L665) is missing `clientId: string`, as described on [the documentation](https://auth0.github.io/node-auth0/AuthenticationClient.html#requestChangePasswordEmail). I can fix this if it is an issue, just want to make sure it is actually missing it.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
Hi @saewitz. PR is ready. Please, could you review it? #58624

**Comment by saewitz** [comment]
Thanks @martin-badin. I also learned you can set `clientId` in the `AuthenticationClient` constructor https://auth0.github.io/node-auth0/AuthenticationClient.html.
